src/lib.rs

```diff
@@ -271,6 +251,19 @@ pub async fn run(command: Subcommands) -> Result<(), SoldeerError> {
             }
         }
         Subcommands::Push(push) => {
+            let path = push
+                .path
+                .unwrap_or(get_current_working_dir().to_str().unwrap().to_string());
+            let path_buf = PathBuf::from(&path);
+
+            // Check for sensitive files or directories
+            if utils::check_for_sensitive_files_or_directories_recursive(&path_buf) {
```

Comment:
just import this so you can use it `check_for_sensitive_files_or_directories_recursive`

Reply:
I don't get what you mean here